.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
```

Comment:
- [ ] In Readme, live demo link should be either removed or add a proper link there to meet the readme requirements.